pkgs/development/python-modules/suds/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPyPy,
+  pytestCheckHook,
+  setuptools,
+  six,
+}:
+
+buildPythonPackage rec {
+  pname = "suds";
+  version = "1.1.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-HVz6dBFxk7JEpCM/JGxIPZ9BGYtEjF8UqLrRHE9knys=";
+  };
+
+  build-system = [ setuptools ];
+
+  nativeBuildInputs = [ six ];
+  nativeCheckInputs = [ pytestCheckHook ];
```

```suggestion
  nativeCheckInputs = [
    pytestCheckHook
    six
  ];
```
Sorry, my mistake. Please add `six` to `nativeCheckInputs`.